src/crc64.c

```diff
@@ -28,6 +28,7 @@
 
 #include "crc64.h"
 #include "crcspeed.h"
+#include "serverassert.h"
```

Comment:
It's hurts a bit to see this, since the CRC files were independent from the rest of the code base until now.

I see all other copied libraries have this though, so I guess it's the way it's done in this code base.

Was that your conclusion?

Reply:
Yeah. It's the way we override the normal "assert" with the valkey server assert, so we get our nice pretty stack traces :). Otherwise it just exits (or until very recently, it was just ignoring it).

Reply:
If anyone wants to use it outside of valkey, it's easy enough to replace this line with a regular `<assert.h>`.